protocol/testutil/keeper/perpetuals.go

```diff
@@ -236,6 +239,7 @@ func CreateNPerpetuals(
 			int32(i),             // AtomicResolution
 			defaultFundingPpm,    // DefaultFundingPpm
 			allLiquidityTiers[i%len(allLiquidityTiers)].Id, // LiquidityTier
+			types.PerpetualMarketType_PERPETUAL_MARKET_TYPE_CROSS, // MarketType
```

Comment:
nit: Alternate between cross and isolated / create at least 1 isolated perpetual.

Reply:
not sure I understand the comment. Doesn't this test need all markets to be cross?

Reply:
I don't think this is a test, it's a test util function? If you change this to be say CROSS for half of the N perps and ISOLATED for the other half what tests break?